co-simulation/bundle/src/assembly/resources/fed/ns3/ns3_installer.sh

```diff
@@ -477,17 +487,18 @@ log "Preparing installation..."
 check_required_programs "${required_programs[*]}"
 check_directory
 
-download_ns3
-
-download_federate
+# download_federate
 
 download_premake5
 
-log "Extracting "$ns3_filename"..."
-extract_ns3 "$ns3_filename" .
+clone_ns3_federate
+
+mkdir "${ns3_installation_path}/ns-allinone-${ns3_version}"
+
+clone_ns3_cv2x
```

Comment:
Have you confirmed that the folders these are getting cloned into is picked up in the build? Like seeing the fabian-eckermann build output?

Reply:
Yes, the cloned folders are picked up in the build. the build output can be seen during the docker image build process